cmds/dhclient/dhclient.go

```diff
@@ -14,64 +14,55 @@
 package main
 
 import (
-	"bytes"
 	"crypto/rand"
 	"flag"
 	"fmt"
-	"io/ioutil"
 	"log"
 	"net"
 	"os"
 	"time"
 
 	"github.com/d2g/dhcp4"
 	"github.com/d2g/dhcp4client"
+	"github.com/mdlayher/dhcp6"
+	"github.com/u-root/u-root/cmds/dhclient/dhcp6client"
 	"github.com/vishvananda/netlink"
 )
 
 const (
-	defaultIface = "eth0"
+	defaultIfname = "enp0s25"
```

Comment:
at this point eth0 is far more common. In the original dhcp I allowed regular expressions. I think at least we should allow a glob pattern. But that said I think that eth0 is a better default choice.